app/docs/getting-started/Introduction/Introduction.mdx

```diff
@@ -31,12 +32,17 @@ Replace `my-project` with your preferred project name.
 
 `Step 3:` Install Keep React library
 
+Install with Npm
 <CopyCode code={keepNpmInstall} />
 
-Or
+Install with Yarn
 
 <CopyCode code={keepYarnInstall} />
 
+Install with Pnpm
```

Comment:
Great, Can you please add this code in NextJs Application Installation Process too?

Reply:
NextJs Application Installation Process updated.